shell/components/LandingPagePreference.vue

```diff
@@ -120,7 +120,7 @@ export default {
             :val="false"
             :value="afterLoginRoute=== 'home' || afterLoginRoute === 'last-visited'"
             :v-bind="$attrs"
-            @update:value="afterLoginRoute = false"
+            @update:value="afterLoginRoute = routeFromDropdown.value"
```

Comment:
We weren't automatically selecting the first cluster when selecting the cluster option.